README.md

```diff
@@ -1492,6 +1492,12 @@ which will halt execution.
 
 [BLAKE3]: https://github.com/BLAKE3-team/BLAKE3/
 
+#### Random
+
+- `pick(len)`<sup>master</sup> - Generate a string of `len` random lowercase hexadecimal digit characters.
+- `pick(len, alphabet)`<sup>master</sup> - Generate a random string of length `len` consisting of characters found in `alphabet`.
```

Comment:
If you're into it, you could add some examples of the use to [the `examples` folder](https://github.com/casey/just/tree/master/examples) -- maybe under the existing kitchen sink ones?

Reply:
I think it's fine not to. I don't really require adding examples for every new function. An example can be added to the readme though.